src/sql/src/session/vars.rs

```diff
@@ -1715,6 +1708,12 @@ feature_flags!(
         "jemalloc heap memory profiling",
         false
     ),
+    (
+        enable_comment,
+        "Enables the COMMENT ON feature for objects in the database (Materialize).",
+        false, // default false
+        false  // internal false
+    )
```

Comment:
cc @ParkMyCar @def- Made this into a feature flag

Reply:
Actually let me separate that out in a different PR, it's not exactly related to the changes here and there are more tests failing.

Reply:
WIP https://github.com/MaterializeInc/materialize/pull/22242